services/movie.py

```diff
@@ -0,0 +1,38 @@
+from db.models import Movie
+
+from django.db.models import QuerySet
+from typing import List
+
+
+def get_movies(
+        genres_ids: List[int] = None,
+        actors_ids: List[int] = None
+) -> QuerySet:
+
+    movies = Movie.objects.all()
+    if genres_ids is not None:
+        movies = movies.filter(genres__id__in=genres_ids)
+    if actors_ids is not None:
+        movies = movies.filter(actors__id__in=actors_ids)
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: List[int] = None,
+        actors_ids: List[int] = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contain string elements and can be None has typehint Optional[list[str]]